frontend/src/lib/components/Subscriptions/SubscriptionsModal.tsx

```diff
@@ -16,24 +19,38 @@ export interface SubscriptionsModalProps extends SubscriptionBaseProps {
 export function SubscriptionsModal(props: SubscriptionsModalProps): JSX.Element {
     const { visible, closeModal, dashboardId, insightShortId, subscriptionId } = props
     const { push } = useActions(router)
+    const { hasAvailableFeature } = useValues(userLogic)
 
     return (
         <>
             <LemonModal onCancel={closeModal} afterClose={closeModal} visible={visible} width={650}>
-                {!subscriptionId ? (
-                    <ManageSubscriptions
-                        insightShortId={insightShortId}
-                        dashboardId={dashboardId}
-                        onCancel={closeModal}
-                        onSelect={(id) => push(urlForSubscription(id, props))}
-                    />
+                {hasAvailableFeature(AvailableFeature.SUBSCRIPTIONS) ? (
+                    !subscriptionId ? (
+                        <ManageSubscriptions
+                            insightShortId={insightShortId}
+                            dashboardId={dashboardId}
+                            onCancel={closeModal}
+                            onSelect={(id) => push(urlForSubscription(id, props))}
+                        />
+                    ) : (
+                        <EditSubscription
+                            id={subscriptionId}
+                            insightShortId={insightShortId}
+                            dashboardId={dashboardId}
+                            onCancel={() => push(urlForSubscriptions(props))}
+                            onDelete={() => push(urlForSubscriptions(props))}
+                        />
+                    )
                 ) : (
-                    <EditSubscription
-                        id={subscriptionId}
-                        insightShortId={insightShortId}
-                        dashboardId={dashboardId}
-                        onCancel={() => push(urlForSubscriptions(props))}
-                        onDelete={() => push(urlForSubscriptions(props))}
+                    <PayGatePage
+                        featureKey={AvailableFeature.SUBSCRIPTIONS}
+                        header={
+                            <>
+                                Introducing <span className="highlight">Subscriptions</span>!
+                            </>
+                        }
+                        caption="Get regular Insight or Dashboard reports directly to your inbox!"
```

Comment:
```suggestion
                        caption="Get Insight or Dashboard reports directly to your inbox!"
```

regular felt odd and most people will know what a subscription is?